### ISSUE TITLE
WP Footer Is Not Visible at 400% Zoom

### ISSUE THREAD

**Issue by MajoMaldonado** [equalizedigital/accessibility-checker#1755]
Location / URL: https://edac-vpat.instawp.co/wp-admin/admin.php?page=accessibility_checker_settings 
Screenshot: 

<img width="772" height="397" alt="Image" src="https://github.com/user-attachments/assets/ff1ab4e1-649f-4e71-8bb0-7b0d622b9171" />

WCAG S.C: 1.4.10 Reflow (Level AA)
Issue Description: Preconditions: Set the display resolution to 1920×1080 and the display scale to 100% in the operating system settings. Then, set the browser zoom level to 400%.

At 400% browser zoom, the "Want to do more with Accessibility Checker? Unlock Pro Features" section and the Version 7.0 information are no longer visible. All page content should remain visible and accessible when viewed at 400% zoom.

**Comment by amberhinds** [comment]
This is a WordPress core issue. I opened a Trac ticket: https://core.trac.wordpress.org/ticket/65488#ticket
